swan-lake/integration-tools/edi-tool.md

```diff
@@ -61,339 +61,238 @@ The command options that are available with the tool are listed below.
-        }
-    ],
-    "A_2000_Loop": [
-        {
-            "Insured_Benefit": {
-                "Yes_No_Condition_or_Response_Code": "Y",
-                "Individual_Relationship_Code": "18",
-                "Maintenance_Type_Code": "025",
-                "Benefit_Status_Code": "A",
-                "Employment_Status_Code": "FT"
-            },
-            "Reference_Information_2": [
-                {
-                    "Reference_Identification_Qualifier": "0F",
-                    "Reference_Identification": "202443307"
-                },
-                {
-                    "Reference_Identification_Qualifier": "1L",
-                    "Reference_Identification": "123456001"
-                }
-            ],
-            "Date_or_Time_or_Period_2": [],
-            "A_2100_Loop": [
-                {
-                    "Individual_or_Organizational_Name": {
-                        "Entity_Identifier_Code": "IL",
-                        "Entity_Type_Qualifier": "1",
-                        "Name_Last_or_Organization_Name": "SMITH",
-                        "Name_First": "WILLIAM",
-                        "Identification_Code_Qualifier": "ZZ",
-                        "Identification_Code": "202443307"
-                    },
-                    "Employment_Class": [],
-                    "Monetary_Amount_Information_2": [],
-                    "Health_Care_Information_Codes": [],
-                    "Language_Use": []
-                }
-            ],
-            "A_2200_Loop": [],
-            "A_2300_Loop": [
-                {
-                    "Health_Coverage": {
-                        "Maintenance_Type_Code": "025",
-                        "Insurance_Line_Code": "DEN"
-                    },
-                    "Date_or_Time_or_Period_4": [
-                        {
-                            "Date_Time_Qualifier": "348",
-                            "Date_Time_Period_Format_Qualifier": "D8",
-                            "Date_Time_Period": "20020701"
-                        }
-                    ],
-                    "Monetary_Amount_Information_3": [],
-                    "Reference_Information_3": [],
-                    "Identification_Card": [],
-                    "A_2310_Loop": [],
-                    "A_2320_Loop": []
-                }
-            ],
-            "A_2400_Loop": [],
-            "A_2500_Loop": [],
-            "A_2600_Loop": []
-        }
-    ],
-    "Transaction_Set_Trailer": {
-        "Number_of_Included_Segments": 12,
-        "Transaction_Set_Control_Number": "12345"
-    }
-}
-```
+    Creating bala
+        target/bala/citymart-porder-any-0.1.0.bala
+    ```
+
+5. Push it to a repository.
+
+    >**Tip:** You can push either to the local repository or the remote repository in [Ballerina Central](https://central.ballerina.io/). Then, any Ballerina project can import this package and use it to work with the EDI files related to purchase orders.
```

Comment:
Is it meaningful to say `remote repository in Ballerian central`?

Reply:
Updated.